recipes/triqs_tprf/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "2.2.0" %}
+
+package:
+  name: triqs_tprf
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/tprf/releases/download/{{ version }}/tprf-{{ version }}.tar.gz
+  sha256: 87389c71bdf690e05b72a5cd7dd2f4f7d8a8ac69b15f1481de1960e7740efc9f
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
```

Comment:
This is a Python 2 only recipe?

Reply:
Yes! This application is Python 2 only.
We are currently working on the Python 3 port which will be the upcoming release.

Reply:
Ok!